client/src/pages/SignUp.js

```diff
@@ -86,6 +101,7 @@ class SignUpForm extends Component {
                 className="FormField__Checkbox"
                 type="checkbox"
                 name="hasAgreed"
+                required="true"
```

Comment:
change the 
`required="true"`
to either
`required` or `required={true}`

The reason being, `"true"` becomes a string and no longer represents a Bool value